deploy/gcp/README.md

```diff
@@ -0,0 +1,99 @@
+# Deploy TiDB Operator and TiDB cluster on GCP GKE
+
+## Requirements:
+* [gcloud](https://cloud.google.com/sdk/install)
+* [terraform](https://www.terraform.io/downloads.html)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl) >= 1.11
+* [helm](https://github.com/helm/helm/blob/master/docs/install.md#installing-the-helm-client) >= 2.9.0
+* [jq](https://stedolan.github.io/jq/download/)
+
+## Configure gcloud
+
+https://cloud.google.com/sdk/docs/initializing
+
+## Setup
+
+The default setup will create a new VPC, two subnetworks, and an f1-micro instance as a bastion machine. The GKE cluster is created with the following instance types as worker nodes:
+
+* 3 n1-standard-4 instances for PD
+* 3 n1-highmem-8 instances for TiKV
+* 3 n1-standard-16 instances for TiDB
+* 3 n1-standard-2 instances for monitor
+
+> *NOTE*: The number of nodes created depends on how many availability zones there are in the chosen region. Most have 3 zones, but us-central1 has 4. See https://cloud.google.com/compute/docs/regions-zones/ for more information.
+
+> *NOTE*: The default setup, as listed above, will exceed the default CPU quota of a GCP project. To increase your project's quota, please follow the instructions [here](https://cloud.google.com/compute/quotas). The default setup will require at least 91 CPUs, more if you need to scale out.
+
+The terraform script expects three environment variables. You can let Terraform prompt you for them, or `export` them ahead of time. If you choose to export them, they are:
+
+* `TF_VAR_GCP_CREDENTIALS_PATH`: Path to a valid GCP credentials file
+* `TF_VAR_GCP_REGION`: The region to create the resources in, for example: `us-west1`
+* `TF_VAR_GCP_PROJECT`: The name of the GCP project
+
+It is generally considered a good idea to create a service account to be used by Terraform. See https://cloud.google.com/iam/docs/creating-managing-service-accounts for more information on how to manage them.
```

Comment:
Suggest moving the service account descriptions after the "TF_VAR_GCP_CREDENTIALS_PATH" part, since they describe how to get a credential file.
BTW, it does not make it clear how to get the credential file.